code/game/turfs/open/misc.dm

```diff
@@ -5,8 +5,8 @@
 	name = "coder/mapper fucked up"
 	desc = "report on github please"
 
-	flags_1 = NO_SCREENTIPS_1
-	turf_flags = CAN_BE_DIRTY_1 | IS_SOLID | NO_RUST
```

Comment:
for the record this does nothing on this subtype because the only time we check for `CAN_BE_DIRTY_1` is turf code in the `MakeDirty()` proc which only exists on `/turf/open/floor`... so I don't know if I should just yeet it here or leave it in case someone wants to leverage it in the future. Thoughts?